packages/web-components/src/components/ic-breadcrumb-group/test/basic/__snapshots__/ic-breadcrumb-group.spec.ts.snap

```diff
@@ -93,7 +93,7 @@ exports[`ic-breadcrumb-group should render collapse on already small devices: sh
       </div>
     </mock:shadow-root>
   </ic-breadcrumb>
-  <ic-breadcrumb class="back show" href="/breadcrumb-4" page-title="Breadcrumb 4" role="listitem" show-back-icon="">
+  <ic-breadcrumb class="back show" href="/breadcrumb-4" page-title="Breadcrumb 4" role="listitem" show-back-icon="true">
```

Comment:
has this change been caused by the upgrade??

Reply:
Yeah it wouldn't pass without changing it to true. I think this is the correct way anyway as we always add "true" to the guidance site, it's only the react ones that build correctly without explicitly setting it to true

Reply:
thanks, was just puzzled why a snapshot had changed, when the code hasn't